### DISCUSSION TITLE
Questions about API Usage

### DISCUSSION THREAD

**Discussion by alexanderdidio** [GriefPrevention/GriefPrevention#2321] [Q&A]
1.) Is x1 and z1 in dataStore#resizeClaim the greater or lesser corner boundary?
2.) When using dataStore#createClaim, does GP handle claim overrides/nearby claims?

**Comment by Jikoo** [comment]
> 1.) Is x1 and z1 in dataStore#resizeClaim the greater or lesser corner boundary?

Yes.
https://github.com/GriefPrevention/GriefPrevention/blob/14ce8e6001a04a434590d2cca787da8dc603ada1/src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java#L871-L903

> 2.) When using dataStore#createClaim, does GP handle claim overrides/nearby claims?

Conflicts? Yes. Nearby? No. If it's unclaimed, it can be claimed.
https://github.com/GriefPrevention/GriefPrevention/blob/14ce8e6001a04a434590d2cca787da8dc603ada1/src/main/java/me/ryanhamshire/GriefPrevention/DataStore.java#L945-L966
